### PR TITLE
Linux chapter manipulating files

### PR DESCRIPTION
**Pull request by redfast00**

### CODE REVIEW

File: intro-Linux/ch_navigating.tex [hunk 1]
```diff
@@ -168,31 +169,31 @@ \section{Permissions}
...
-To look for files using incomplete names, you can use a wildcard ``*''; note
-that you need to escape the ``*'' to avoid that Bash \emph{expands} it into
-``afile.txt'':
+To look for files using incomplete names, you can use a wildcard \verb|*|; note
+that you need to escape the \verb|*| to avoid that Bash \emph{expands} it into
+\verb|afile.txt| by adding double quotes:
 
 \begin{prompt}
-%\shellcmd{find . -name '*.txt'}%
+%\shellcmd{find . -name "*.txt"}%
```
Comment (boegel) [inline intro-Linux/ch_navigating.tex]: Nice catch here :)

**Comment by boegel** [comment]
Looks all good to me, let's bump the version & get it merged?

**Comment by redfast00** [comment]
I'd like to first get #216 merged, so building this won't take forever

**Comment by boegel** [comment]
@redfast00 #216 merged

**Pull request merged**